admin/app/templates/authenticated/tools.hbs

```diff
@@ -11,17 +11,27 @@
         <h2 class="page-section__title">Référentiel</h2>
       </header>
       <div>
-        <p>Une version du référentiel de données pédagogique est créé automatiquement quotidiennement (vers 4h00) et le référentiel utilisé par l'application est mis à jour (vers 6h00).</p>
+        <p>Une version du référentiel de données pédagogique est créé quotidiennement (vers 4h00) et le référentiel utilisé par l'application est mis à jour (vers 6h00).</p>
         <p>Si le cache a été vidé, il peut s’avérer utile ou nécessaire de recharger le référentiel.</p>
         <div class="callout alert"><strong>Attention !</strong> Le rechargement du référentiel est une opération risquée. Il est recommandé de ne l’effectuer qu’en cas de force majeure, accompagné d’un développeur ou d'une développeuse.</div>
         <div class="callout info">Durée de l’opération : <strong>≈10s</strong>.</div>
+
         <PixButton
           @triggerAction={{fn this.refreshLearningContent}}
         >
           <FaIcon @icon="sync"></FaIcon> Recharger le cache
         </PixButton>
+        <br><br>
+        <p>Si quelque chose a été changé dans le référentiel et qu'il faut appliquer ces changements dans l'application, il faut créer une nouvelle version et mettre à jour le cache.</p>
+        <div class="callout info">Durée de l’opération : <strong>≈1mn</strong>.</div>
+        <PixButton
+          @triggerAction={{fn this.createLearningContentReleaseAndRefreshCache}}
```

Comment:
**suggestion:** `@triggerAction={{this.createLearningContentReleaseAndRefreshCache}}`